src/components/CheckResultDetails.tsx

```diff
@@ -98,20 +102,40 @@ export function CheckResultDetails ({ checkResult, combinedStatus }: Props) {
           overflow: 'hidden',
         }}
       >
-        {CheckResultIcon}
-        <Typography style={{marginLeft: '10px'}} className={styles.coloredText}>
-          {checkResult.check.name}
-          
-          <span style={{fontSize: "smaller"}} className={styles.coloredSubtext}> &bull; <b>{checkResult.check.category?.name || "Uncategorized"}</b> check</span>
-        </Typography>
+        <Box style={{ display: "flex", alignItems: "center" }}>
+          <div className={styles.checkResultIcon} style={{ color: resultColorMap[combinedStatus].color}}>
+            { checkResultIcons[combinedStatus] }
+          </div>
+          <Typography className={styles.coloredText}>
+            {checkResult.check.name}
+
+            {checkResult.check.category && <>
+              <span className={styles.separator}>&#65372;</span>
+              <span className={styles.coloredSubtext}>
+                <TableOutlined className={styles.checkResultIcon} />
```

Comment:
this probably creates merge issues because scorecards use a different symbol in the app. what's our plan to tackle this?

Reply:
I think scorecards are in now so this can probably be done in this MR